aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -155,6 +163,18 @@ module aptos_framework::fungible_asset {
         frozen: bool,
     }
 
+    fun default_to_concurrent_fungible_supply(): bool {
+        features::concurrent_fungible_assets_enabled()
+    }
+
+    fun default_to_concurrent_fungible_balance(): bool {
+        features::concurrent_fungible_assets_enabled()
+    }
+
+    fun auto_upgrade_to_concurrent_fungible_balance(): bool {
+        features::concurrent_fungible_assets_enabled()
+    }
```

Comment:
```suggestion

    inline fun default_to_concurrent_fungible_supply(): bool {
        features::concurrent_fungible_assets_enabled()
    }

    inline fun default_to_concurrent_fungible_balance(): bool {
        features::concurrent_fungible_assets_enabled()
    }

    inline fun auto_upgrade_to_concurrent_fungible_balance(): bool {
        features::concurrent_fungible_assets_enabled()
    }
```


How about making these all inline to reduce function calls on stack? 

The gas schedule charges for each function call, and these can easily be inlined

Since these are inner function for lots of functions across the framework stack, failing to optimizing them as much as possible here will have adverse consequences downstream